### ISSUE TITLE
import fails with latest version of pandas

### ISSUE THREAD

**Issue by marketneutral** [sbebo/joypy#46]
`import joypy`  throws

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-47a753585c6f> in <module>
----> 1 import joypy
      2 import numpy as np
      3 import pandas as pd
      4 import matplotlib.pyplot as plt
      5 from sklearn.linear_model import LinearRegression

~/.local/lib/python3.6/site-packages/joypy/__init__.py in <module>
      3 __version__ = '0.1.10'
      4 
----> 5 from .joyplot import joyplot, plot_density, _joyplot

~/.local/lib/python3.6/site-packages/joypy/joyplot.py in <module>
      2 
      3 import numpy as np
----> 4 from pandas.plotting._tools import (_subplots, _flatten)
      5 import os
      6 import matplotlib as mpl

ModuleNotFoundError: No module named 'pandas.plotting._tools'
```

**Comment by leotac** [comment]
Hi, I think it's the same issue in #38. It should be fixed in the latest version (0.2.1).

**Comment by marketneutral** [comment]
Yes, that's right. Thank you! It works with 0.2.1.
